frontend/packages/kubevirt-plugin/src/hooks/use-ssh-command.ts

```diff
@@ -2,7 +2,7 @@ import { getCloudInitValues, TARGET_PORT } from '../components/ssh-service/SSHFo
 import { VMIKind } from '../types';
 import { useSSHService2 } from './use-ssh-service';
 
-export type useSSHCommandResult = {
+type UseSSHCommandResult = {
```

Comment:
Types not used outside the module whose name violates the ESLint naming rule were renamed and the `export` keyword was removed.